src/styles/_FacetsEditor.scss

```diff
@@ -3,7 +3,8 @@
   padding-right: 0;
 }
 
-.facets__isEdited {
+.facets__isEdited,
```

Comment:
curious about the naming convention that uses `__`. How should this be interpreted vs camelCase and dashed-names? We currently have all three conventions.

Reply:
@jeffmcaffer this is coming from the BEM naming conventions (http://getbem.com/naming/). 

The issue I see in the above code is that __ should be used for `block__elem` while `isEdited` is more of a modifier.

Hence, @storrisi  let's ensure to conver this (and any other modified) to `facets--isEdited` etc .

@jeffmcaffer can you confirm we are happy to use BEM?

Reply:
I've fixed this, the right BEM convention is with double dash --

Reply:
I'm fine with BEM (don't know much about this space). One thing we should IMHO avoid is an explosion of "duplicate" styles. I've seen cases where there were 100s of styles with many basically being the same as one another but on a different page of the UI. It made it nearly impossible to get the UX consistent.

More generally, please do bring more coherence and consistency to the code! This would be a great next step once the current efforts for editing etc are done.